src/App.tsx

```diff
@@ -1,12 +1,84 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { Good } from './react-app-env';
 import './App.scss';
+import { Goods } from './components/Goods/Goods';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
+  const [unactive, setUnactive] = useState({
+    unactiveAll: false,
+    unactive5: false,
+    unactiveRed: false,
+  });
```

Comment:
it isn't a good idea to create a key for each button, for this task it's ok but it isn't a flexible solution